var/spack/repos/builtin/packages/gdl/package.py

```diff
@@ -41,8 +42,8 @@ class Gdl(CMakePackage):
     depends_on('plplot+wx+wxold', when='+wx@5.12:')
     depends_on('plplot~wx', when='~wx')
     depends_on('proj', when='+proj')
-    depends_on('py-numpy', type=('build', 'run'), when='+python')
-    depends_on('python@2.7:2.8', type=('build', 'run'), when='+python')
+    depends_on('py-numpy', type=('build', 'run'), when='+embed-python')
+    depends_on('python@2.7:2.8', type=('build', 'run'), when='+embed-python')
```

Comment:
```suggestion
    extends('python@2.7:2.8', when='+embed-python')
```
This has the advantage that you can run `spack activate gdl` and have it symlink the Python libraries to the Python installation prefix.

Reply:
This is actually already done (for the `python` variant), see line 34.
The `python_embed` variant doesn't build a Python library, it allows calling Python from within GDL.